### ISSUE TITLE
Fix S2302 FN: Issue is not raised when the name of one of the method parameters is a raw string literal

### ISSUE THREAD

**Issue by mary-georgiou-sonarsource** [SonarSource/sonar-dotnet#6441]
### Description

Rule S2302 does not raise an issue when the string in the throw statement contains the name of one of the method parameters
but in a raw string literal.

### Repro steps

```csharp
public void Method_With_RawStringLiterals(int arg1)
{
    if (arg1 < 0)
        throw new Exception("""arg1""");  // FN
}
```

### Related information

* C#/VB.NET Plugins version: 8.49
